pyproject.toml

```diff
@@ -20,6 +20,8 @@ dependencies = [
     "astropy",
     "citation-compass>=0.0.3",
     "dust_extinction",
+    "healpy",
```

Comment:
This raises a small alarm for me, because I know LSDB has been working to remove healpy dependency this past year. 

I believe in part it is due to installation difficulties across multiple platforms, but in part it's because [healpy has a GPL license](https://github.com/healpy/healpy?tab=GPL-2.0-1-ov-file). Is this alright for TDAstro?

Reply:
Let me look into this a bit.

Reply:
I'm only using healpy for testing and plotting. I removed both of those and switched to cdshealpix.